### PR TITLE
Local filesystem to Google Drive Operator - system tests migration (A…

### PR DESCRIPTION
**Pull request by bkossakowska**
…IP-47)

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: tests/system/providers/google/suite/example_local_to_drive.py [hunk 1]
```diff
@@ -26,16 +26,26 @@
...
+
+SINGLE_FILE_LOCAL_PATHS = [str(Path(LOCAL_PATH) / FILE_NAME_1)]
+MULTIPLE_FILES_LOCAL_PATHS = [str(Path(LOCAL_PATH) / FILE_NAME_1), str(Path(LOCAL_PATH) / FILE_NAME_2)]
+
+DRIVE_FOLDER = "test-folder"
 
 with models.DAG(
-    "example_local_to_drive",
+    DAG_ID,
     start_date=datetime(2021, 1, 1),
```
Comment (bhirsz) [inline tests/system/providers/google/suite/example_local_to_drive.py:44]: schedule=@once missing, without schedule it will not run

**Comment by bhirsz** [comment]
Please also remove old test_<service_name>_system.py file

**Comment by bkossakowska** [comment]
> Please also remove old test_<service_name>_system.py file

no file: test_<service_name>_system.py

File: tests/system/providers/google/suite/example_local_to_drive.py [hunk 2]
```diff
@@ -20,22 +20,37 @@
...
+LOCAL_PATH = str(Path(__file__).parent / "resources")
+
+SINGLE_FILE_LOCAL_PATHS = [str(Path(LOCAL_PATH) / FILE_NAME_1)]
+MULTIPLE_FILES_LOCAL_PATHS = [str(Path(LOCAL_PATH) / FILE_NAME_1), str(Path(LOCAL_PATH) / FILE_NAME_2)]
+
+DRIVE_FOLDER = "test-folder"
+
+os.environ[
+    "AIRFLOW_CONN_GOOGLE_CLOUD_DEFAULT"
+] = "google-cloud-platform://?extra__google_cloud_platform__scope=https://www.googleapis.com/auth/drive"
```
Comment (bhirsz) [inline tests/system/providers/google/suite/example_local_to_drive.py:44]: Should be defined in the environment itself (ie breeze config file)

Comment (bkossakowska) [reply tests/system/providers/google/suite/example_local_to_drive.py]: ok

File: docs/apache-airflow-providers-google/example-dags.rst [hunk 3]
```diff
@@ -17,13 +17,13 @@
 
 Example DAGs
 ============
-
+docs/apache-airflow-providers-google/example-dags.rst
```
Comment (bhirsz) [inline docs/apache-airflow-providers-google/example-dags.rst:20]: Is this line meant to be there?

Comment (bkossakowska) [reply docs/apache-airflow-providers-google/example-dags.rst]: it's possible that tests failed without it before or it was added automatically by pre-commit, 
but now tests passed without it

**Pull request closed without merging**